heidrun/ct_dc.rb

```diff
@@ -0,0 +1,95 @@
+
+# If more than one dc:contributor property exists then it drops the last one
+# If one or zero exist return nil
```

Comment:
Can you elaborate on how it returns `nil` if there's one element?  I may be overlooking something, but it looks like it would return the last element.